cypress/integration/page.spec.js

```diff
@@ -1,8 +1,43 @@
 'use strict';
 
-// TODO: edit it
-describe('Page', () => {
-  it('should be visitable', () => {
+Cypress.Commands.add('clickOnWall',
+  (position, xCoord, yCoord) => {
+    // NOTE: waiting for transition
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.get('.wall').click(position).wait(1000);
```

Comment:
Based on documentation this should not be needed: https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Animations

Reply:
yes, but we need this in that particular case. we have consulted with Yurii

Reply:
discussed that we'll debug how to make it work.